packages/theme-wizard-website/e2e/pages/ThemeWizardPage.ts

```diff
@@ -0,0 +1,52 @@
+import { type Page, type Locator, expect } from '@playwright/test';
+
+export class ThemeWizardPage {
+  readonly preview: Locator;
+  private readonly templateSelect: Locator;
+
+  constructor(public readonly page: Page) {
+    this.preview = this.page.getByTestId('preview');
+    this.templateSelect = this.page.getByLabel('Kies een template');
+  }
+
+  async goto() {
+    await this.page.goto('/');
+    await expect(this.preview).toBeVisible();
+  }
+
+  async selectTemplate(templateName: string) {
+    await this.templateSelect.selectOption({ label: templateName });
+  }
+
+  async changeHeadingFont(fontName: string) {
+    await this.page.getByLabel('Koppen').selectOption({ label: fontName });
+  }
+
+  async changeBodyFont(fontName: string) {
+    await this.page.getByLabel('Lopende tekst').selectOption({ label: fontName });
+  }
+
+  async verifyFontChange(element: Locator, fontFamily: RegExp) {
+    await expect(element).not.toHaveCSS('font-family', fontFamily);
```

Comment:
Ik hou het liefst de meeste `expect`s in de test files zelf, dan kun je bijvoorbeeld custom error messages of timeouts blijven doorgeven. En het meest belangrijk: dan kun je `expect.soft()` gebruiken! (Test stopt dan niet na de eerste failure, zo kun je meerdere assertions in 1 test doen)

Als je echt heel graag custom matchers wil, zou je hiernaar kunnen kijken: https://playwright.dev/docs/test-assertions#add-custom-matchers-using-expectextend

Reply:
Wat je zou kunnen doen is bv. een `getAppliedFont()` oid zodat je `expect(getAppliedFont(previewPage.getHeading(2))).toBe(/MyFont/)` kunt doen. Goed compromis?